.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1330,62 +1330,75 @@ For each of the following cases, create a moderated session with the user using
 
 ## Performance
 
-### Scaling Test
-Scale up the number of nodes/clusters a few times for each configuration below.
+For all performance tests
 
  1) Verify that there are no memory/goroutine/file descriptor leaks
  2) Compare the baseline metrics with the previous release to determine if resource usage has increased
- 3) Restart all Auth instances and verify that all nodes/clusters reconnect
 
- Perform reverse tunnel node scaling tests for all backend configurations:
-  - [ ] etcd - 10k
-  - [ ] DynamoDB - 10k
-  - [ ] Firestore - 10k
-  - [ ] Postgres - 10k
+### Ansible-like Test
+
+Run the [ansible-like](https://github.com/gravitational/teleport/tree/4fd411add0c6fa7d4d0d19b1cf0c5c13c541498e/assets/loadtest/ansible-like)
+test against a Cloud tenant with 60k nodes dispersed geographically in multiple regions.
+
+ - [ ] DynamoDB
+ - [ ] CRDB
+
+### Simluated load test
 
-  Perform the following additional scaling tests on DynamoDB:
- - [ ] 10k direct dial nodes.
- - [ ] 500 trusted clusters.
+Run a simulated 30k load test against all self-hosted only backends by running the following command
+simultaneously from two auth instances.
+
+```shell
+tctl loadtest node-heartbeats --duration=30m --count=5000 --ttl=2m --interval=1m --labels=2 --concurrency=32
```

Comment:
Isn't this going to be 10k? Should the count be 15000 for 30k total fake agents?